File: CS/02_Text/ExtractTextFromSpecificArea.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;
using Spire.Pdf.Texts;
using System.IO;
namespace ExtractTextFromSpecificArea
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

         private void button1_Click(object sender, EventArgs e)
        {
            // Specifying the path to the source PDF file
            string input = @"..\..\..\..\..\..\Data\ExtractTextFromSpecificArea.pdf";

            // Load the PDF file
            PdfDocument pdf = new PdfDocument();
            pdf.LoadFromFile(input);

            // Get the first page of the PDF
            PdfPageBase page = pdf.Pages[0];

            // Define options for text extraction
            PdfTextExtractOptions options = new PdfTextExtractOptions();
            options.ExtractArea = new RectangleF(80, 180, 500, 200);

            // Create a PdfTextExtractor object and extract text using the specified options
            PdfTextExtractor pdfTextExtractor = new PdfTextExtractor(page);
            string text = pdfTextExtractor.ExtractText(options);

            // Save the extracted text to a .txt file
            StringBuilder sb = new StringBuilder();
            sb.AppendLine(text);
            string result = "ExtractText_result.txt";
            File.WriteAllText(result, sb.ToString());

            Viewer(result);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/02_Text/ExtractTextFromSpecificArea.vb
Imports System.Text
Imports Spire.Pdf
Imports Spire.Pdf.Texts
Imports System.IO

Namespace ExtractTextFromSpecificArea
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' Specify the input file path
            Dim input As String = "..\..\..\..\..\..\Data\ExtractTextFromSpecificArea.pdf"

            ' Create a new instance of PdfDocument
            Dim pdf As New PdfDocument()

            ' Load the PDF file from the specified input path
            pdf.LoadFromFile(input)

            ' Get the first page of the document
            Dim page As PdfPageBase = pdf.Pages(0)

            ' Create PdfTextExtractOptions and set the extraction area using a rectangle
            Dim options As PdfTextExtractOptions = New PdfTextExtractOptions
            options.ExtractArea = New RectangleF(80, 180, 500, 200)

            ' Create a PdfTextExtractor instance for the page
            Dim pdfTextExtractor As PdfTextExtractor = New PdfTextExtractor(page)

            ' Extract the text from the specified area using the options
            Dim text As String = pdfTextExtractor.ExtractText(options)

            ' Create a StringBuilder instance
            Dim sb As New StringBuilder()

            ' Append the extracted text to the string builder
            sb.AppendLine(text)

            ' Specify the output file path
            Dim result As String = "ExtractText_result.txt"

            ' Write the contents of the string builder to the output file
            File.WriteAllText(result, sb.ToString())

            ' Close the document
            pdf.Close()

            ' Launch the txt file
            Viewer(result)
        End Sub
        Private Sub Viewer(ByVal fileName As String)
            Try
                Process.Start(fileName)
            Catch
            End Try
        End Sub
    End Class
End Namespace
